lib/cli.js

```diff
@@ -121,6 +123,25 @@ function getWatcher(isWatching) {
   return isWatching ? broccoli.Watcher : require('./dummy-watcher');
 }
 
+function buildWatcherOptions(options) {
+  if (!options) {
+    options = {};
+  }
+  const watcher = options.watcher;
+
+  if (watcher && ['polling', 'watchman', 'node', 'events'].indexOf(watcher) === -1) {
+    throw new Error(`Unknown watcher type --watcher=[polling|watchman|node] but was: ${watcher}`);
+  }
+
+  return {
+    saneOptions: {
+      poll: watcher === 'polling',
+      watchman: watcher === 'watchman' || watcher === 'events' || !watcher,
```

Comment:
I think we may want to consider dropping `events` as `watchman` is both clear and concise and already covers that one.

Reply:
If there is no watcher specified,  it likely isn't safe to just assume `watchman` (that would also be backwards incompatible).

We should likely either default to `node` or use `execSync` to see if the `watchman version` returns some minimum version.

Reply:
seems also ci detected the problem. fixing it now